client/src/components/LoginForm.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from "react";
+import { Button, Form, Grid, Header, Icon, Segment } from "semantic-ui-react";
+
+const LoginForm = () => (
+  <div className="login-form">
+    {/*
+      All the elements up to the `Grid`
+      below must have a height of 100%.
+    */}
+    <style>{`
+      body > div,
+      body > div > div,
+      body > div > div > div.login-form {
+        height: 100%;
+      }
+    `}</style>
```

Comment:
Looks good! If you want you can move this to a separate CSS file and import it like you did with `semantic-ui-css/semantic.min.css`